auslib/AUS.py

```diff
@@ -35,23 +46,18 @@ def getFallbackChannel(channel):
 class AUSRandom:
     """Abstract getting a randint to make it easier to test the range of
     possible values"""
-
-    def __init__(self, min=0, max=99):
-        self.min = min
-        self.max = max
+    min = 0
+    max = 99
 
     def getInt(self):
         return randint(self.min, self.max)
 
-    def getRange(self):
-        return range(self.min, self.max + 1)
-
 
 class AUS:
 
-    def __init__(self):
+    def __init__(self, rand=None):
         self.specialForceHosts = None
-        self.rand = AUSRandom()
+        self.rand = rand or AUSRandom().getInt
```

Comment:
We tend to use mocking instead of dependency injection in Balrog - can you do that here?

Reply:
OK.